### DISCUSSION TITLE
Is there a detection clear event in Frigate?

### DISCUSSION THREAD

**Discussion by my-umd** [blakeblackshear/frigate#5820] [General]
Maybe an answered question.  For motion detection, there is a clear event.  But for object detection, I seems not finding such event?  What I want to do is to set up an automation, when a person is detected in my porch, turn on porch light.  Looks simple, but I don't know when to turn the light off.  Thinking to turn off light when no person is detected.  What is a good event to use?  The occupancy binary sensor?
I see people use a timer.  Do not think it is a good approach.  Thanks.

**Comment by NickM-27** [comment]
Yes the MQTT events topic has an event ended message (type: end)

**Reply by my-umd** [reply]
Thanks for confirming @NickM-27.  Another reason I ask this is that, in Home Assistant, when a person is detected, the snapshot is always shown with an 'active' state, even though the event has been passed for hours (i.e., the person has left).  I am not sure if this is by design.  But it makes me thinking there might not be an appropriate event fired to notify Home Assistant to change the state.

**Reply by NickM-27** [reply]
> Thanks for confirming @NickM-27.  Another reason I ask this is that, in Home Assistant, when a person is detected, the snapshot is always shown with an 'active' state, even though the event has been passed for hours (i.e., the person has left).  I am not sure if this is by design.  But it makes me thinking there might not be an appropriate event fired to notify Home Assistant to change the state.

"Active" on that entity simply means it has an image. It means nothing really.

**Reply by my-umd** [reply]
I see.  Thanks @NickM-27.  Looks I miss understood that label.

**Comment by blakeblackshear** [comment]
I use the automatically created zone sensor for person count to turn on lights and turn them off after it has dropped to 0 for a period of time.

**Reply by my-umd** [reply]
Cool!  Thanks @blakeblackshear. 
 Using person count is a good idea.  I was actually thinking how to deal with a bunch of people :)
